kvstore/sql.go

```diff
@@ -109,18 +123,24 @@ func (o SQL) Get(key string) ([]string, error) {
 		return nil, err
 	}
 
-	// nolint:gosec
-	// o.TableName has been checked by isSafeTblName on init
-	q := fmt.Sprintf("SELECT DISTINCT vals FROM %s WHERE key = ?", o.TableName)
+	query := sq.Select("kv_val").Distinct().
+		From(o.TableName).
+		Where(sq.Eq{"kv_key": key}).
+		PlaceholderFormat(o.Placeholder)
+
+	query_text, args, err := query.ToSql()
```

Comment:
nit: https://go.dev/doc/effective_go#mixed-caps

```suggestion
	queryText, args, err := query.ToSql()
```